client/python/gradio_client/client.py

```diff
@@ -27,6 +27,13 @@ def __init__(
         hf_token: str | None = None,
         max_workers: int = 40,
     ):
+        """
+        Parameters:
+            space: The name of the Space to load, e.g. "abidlabs/pictionary". If it is a private Space, you must provide an hf_token. app. Either `space` or `src` must be provided.
```

Comment:
Not related to your PR but just thought about this - what if we collapse `space` and `src` into the same parameter? Basically we automatically detect if the value is a space name or a url.

Reply:
Sure, I don't think that shouldn't lead to any conflicts so let's do it